kafka/kafka-docker.sh

```diff
@@ -16,6 +18,13 @@
 : ${KAFKA_LOG_CLEANER_ENABLE:=true}
 : ${KAFKA_ZOOKEEPER_CONNECT:=$ZOOKEEPER_PORT_2181_TCP_ADDR:$ZOOKEEPER_PORT_2181_TCP_PORT}
 : ${KAFKA_ZOOKEEPER_CONNECTION_TIMEOUT_MS:=6000}
+: ${KAFKA_AUTO_CREATE_TOPICS_ENABLE:=true}
+: ${KAFKA_DELETE_TOPIC_ENABLE:=true}
```

Comment:
Two questions here.

1. Some of these are non-defaults that may not always be good settings. `delete.topic.enable` is probably fairly safe. `offsets.storage` not using the default may be surprising/confusing to some people.

2. Some of these aren't broker settings? `compression.topics`, `acks` I think?